src/node/rpc/member_frontend.h

```diff
@@ -1024,6 +1024,33 @@ namespace ccf
         .set_auto_schema<jsgov::Proposal, jsgov::ProposalInfoSummary>()
         .install();
 
+      using ProposalID = ccf::MemberId;
+      using AllOpenProposals = std::map<ProposalID, jsgov::ProposalInfo>;
```

Comment:
```suggestion
      using AllOpenProposals = std::map<ProposalId, jsgov::ProposalInfo>;
```

This type already exists. Reusing `MemberId` results in an inaccurate OpenAPI (claiming the map is `MemberId_to_ProposalInfo`). If we want more precise OpenAPI describing a `ProposalId` (ie - constraining it to a fixed length hex string, rather than an arbitrary string), we should replace the existing typedefs of `ProposalId` to use `EntityId` or similar.

Reply:
Yeah, that is the only reason I marked it as Draft. I should have a left a comment here. My bad.

> we should replace the existing typedefs of ProposalId to use EntityId or similar.

I was planning on doing this as well